### PR TITLE
clean code and safely inspect fix.

### PR DESCRIPTION
**Pull request by DogukanKundum**

### CODE REVIEW

**Review by ctubbsii — COMMENTED** [review]
Were these changes generated by a tool? There's a mix of stuff that's nice, but some other stuff that probably shouldn't happen. For one, there's a lot of unnecessary whitespace changes. We haven't applied our project formatter to our examples (yet), so that's probably fine for now, but it's a bit disruptive. We also tend to not suppress all warnings (we prefer to fix them), and also would prefer not to leave in commented-out code.

There's some good stuff in here, though, too... like limiting the visibility for fields, methods, and classes, when they don't need to be as visible.

I'm just not sure it's a great idea to merge in the current state.

**Comment by DogukanKundum** [comment]
Yes of course. It may be best to use the same format. also my goal is to be a developer. There have been changes made according to sonarqube results. We can make the changes by browsing together. be part of my target community.

File: stresso/src/test/java/stresso/ITBase.java [hunk 1]
```diff
@@ -37,99 +37,102 @@
...
-    try (FluoAdmin admin = FluoFactory.newAdmin(config)) {
-      InitializationOptions opts =
-          new InitializationOptions().setClearZookeeper(true).setClearTable(true);
-      admin.initialize(opts);
+
+    void preInit(FluoConfiguration config) {
     }
 
-    config.getAppConfiguration().clear();
+// --Commented out by Inspection START (01.03.2019 15:22):
```
Comment (keith-turner) [inline stresso/src/test/java/stresso/ITBase.java:94]: What is this?

File: stresso/src/main/java/stresso/trie/Constants.java [hunk 2]
```diff
@@ -18,15 +18,15 @@
...
+ *  global constant for fluo-example projects
  */
 public class Constants {
 
-  public static final TypeLayer TYPEL = new TypeLayer(new StringEncoder());
+    public static final TypeLayer TYPEL = new TypeLayer(new StringEncoder());
 
-  public static final Column COUNT_SEEN_COL = new Column("count", "seen");
-  public static final Column COUNT_WAIT_COL = new Column("count", "wait");
+    public static final Column COUNT_SEEN_COL = new Column("count", "seen");
```
Comment (keith-turner) [inline stresso/src/main/java/stresso/trie/Constants.java:27]: Why add the spaces?

File: stresso/src/test/java/stresso/TrieMapRedIT.java [hunk 3]
```diff
@@ -35,102 +28,107 @@
...
 import stresso.trie.Unique;
 
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 /**
  * Tests Trie Stress Test using MapReduce Ingest
  */
+@SuppressWarnings("ALL")
```
Comment (keith-turner) [inline stresso/src/test/java/stresso/TrieMapRedIT.java:38]: Why add this?

Comment (DogukanKundum) [reply stresso/src/test/java/stresso/TrieMapRedIT.java]: my formatter different yours. this line code false. i’ll delete. sorry.

File: stresso/src/main/java/stresso/trie/Constants.java (hunk 2 above)
Comment (DogukanKundum) [reply stresso/src/main/java/stresso/trie/Constants.java:27]: my formatter different yours. this line code false at Constants.class. i’ll delete. sorry.

File: stresso/src/test/java/stresso/TrieMapRedIT.java (hunk 3 above)
Comment (DogukanKundum) [reply stresso/src/test/java/stresso/TrieMapRedIT.java:38]: Intellij add getting at inspected. Only classes, methods, variable declarations, parameters and packages may be annotated. Therefore, you cannot use SuppressWarnings("all") on an if statement.

Of course this line can delete. :)

File: stresso/src/test/java/stresso/ITBase.java (hunk 1 above)
Comment (DogukanKundum) [reply stresso/src/test/java/stresso/ITBase.java:94]: Intellij IDE added auto comment line. I don’t looking check of control repeat. I’ll delete.

**Comment by ctubbsii** [comment]
@DogukanKundum Did you close this because you are no longer interested in contributing? Or because you do not have time to address the questions raised in the review?

**Comment by DogukanKundum** [comment]
> @DogukanKundum Did you close this because you are no longer interested in contributing? Or because you do not have time to address the questions raised in the review?

No. I want to contributor. But, first step i have your codeFormatter.xml. you send me codeFormatter.xml. after, I'll edit once this code changed. No problem bro.

**Pull request closed without merging**